src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import { Table } from "antd";
+import { getNCBIExports } from "../../../apis/ncbi/ncbi";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import NcbiUploadStates from "../upload-states";
+
+export function NcbiExportTable({ url }) {
+  const [exports, setExports] = useState(null);
+
+  useEffect(() => {
+    getNCBIExports({ url }).then(setExports);
+  }, [url]);
+
+  const columns = [
+    {
+      title: i18n("NcbiExportTable.id"),
+      dataIndex: "id",
+      render(id, item) {
+        return (
+          <a className="t-biosample-id" href={`${window.location.href}/${id}`}>
```

Comment:
Is there a better way to set the href then using the window.location.href?

Reply:
Updated in 4af6a7e